metadata-ingestion/tests/performance/test_bigquery_usage.py

```diff
@@ -0,0 +1,96 @@
+import logging
+import os
+import random
+import sys
+from datetime import timedelta
+from typing import Generator
+
+import humanfriendly
+import psutil
+import pytest
+from performance.bigquery import generate_events, ref_from_table
+from performance.data_generation import generate_data, generate_queries
+
+from datahub.ingestion.source.bigquery_v2.bigquery_config import (
+    BigQueryUsageConfig,
+    BigQueryV2Config,
+)
+from datahub.ingestion.source.bigquery_v2.bigquery_report import (
+    BigQueryV2Report,
+    logger as report_logger,
+)
+from datahub.ingestion.source.bigquery_v2.usage import BigQueryUsageExtractor
+from datahub.utilities.perf_timer import PerfTimer
+
+
+def set_log_level(logger: logging.Logger, level: int) -> Generator[None, None, None]:
+    old_log_level = logger.level
+    try:
+        logger.setLevel(level)
+        yield
+    finally:
+        logger.setLevel(old_log_level)
+
+
+@pytest.fixture(autouse=True)
+def default_log_level_error():
+    root_logger = logging.getLogger()
+    stream_handler = logging.StreamHandler(sys.stdout)
+    try:
+        root_logger.addHandler(stream_handler)
+        yield from set_log_level(root_logger, logging.ERROR)
+    finally:
+        root_logger.removeHandler(stream_handler)
+
+
+@pytest.fixture
+def report_log_level_info():
+    yield from set_log_level(report_logger, logging.INFO)
```

Comment:
I didn't see any error logs which made debugging these tests pretty annoying. Is this not usually the case and I'm doing something wrong? I also want to see INFO logs from the report class, so that when you run these tests, you can see how long each stage took